.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
```

Comment:
This should be security update only. Otherwise, the project can get spammed by automatic PRs.

Reply:
The project doesn't have so much deps, I think we could setup weekly or monthly updates at least to keep them always updated

Reply:
It is `daily` right now. 

I am fine with weekly or monthly.